pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -22,7 +24,17 @@ class Range(Module):
   is_original_torch_module = False
 
   def create_returnn_layer_dict(self, limit, start, delta, dtype, sparse=False) -> Dict[str, Any]:
-    return {"class": "range", "limit": limit, "start": start, "delta": delta, "dtype": dtype, "sparse": sparse}
+    if isinstance(limit, SizeValue):
+      limit = Length(axis=limit.axis)(limit.tensor)
```

Comment:
You should never pass `axis` as integer.

Reply:
Ah sorry, I was confused. This is here on PyTorch level.

Reply:
But anyway. It depends a bit on how we design `Length`. If this can get a `Dim`, we can simply have here:
```suggestion
      limit = Length(axis=limit.dim_tag)(limit.tensor)
```

Reply:
As we might need this code maybe in multiple places, and also as we maybe will rewrite this later, I would make a helper function here:
```suggestion
      limit = _size_value_to_tensor(limit)
```
Or maybe as a method of `SizeValue`:
```suggestion
      limit = limit.as_tensor()
```

Reply:
When implementing it as a method of `SizeValue`, I need to import `Length` in `_C.py` which also leads to a circular import. I could define `_size_value_to_tensor` in this file (or possibly there is a better place?).